template/create.tmpl

```diff
@@ -2,16 +2,19 @@
 	b := h.client.{{ $.Name }}.Create()
 	// Add all fields.
 	{{- range $f := $.Fields }}
-		{{- if $f.Optional }}
-			if v, ok := req.{{ $f.StructField}}.Get(); ok {
-				{{- $x := "v" }}
-				{{- if $f.IsEnum }}{{ $x = convertTo $f.Type.String $x }}{{ end }}
-				b.Set{{ $f.StructField }}({{ $x }})
-			}
-		{{- else }}
-			{{- $x := print "req." $f.StructField }}
-			{{- if $f.IsEnum }}{{ $x = convertTo $f.Type.String $x }}{{ end }}
-			b.Set{{ $f.StructField }}({{ $x }})
+	    {{- $a := fieldAnnotation $f }}
+	    {{- if or (not $a) (not $a.Field.ReadOnly) }}
```

Comment:
Please apply De'Morgan here :wink:

Reply:
yes. was lazy there. DeMorgan's applied. should be ready to merge now